### DISCUSSION TITLE
importAllFragmentsFrom multiple packages

### DISCUSSION THREAD

**Discussion by alexvcasillas** [dotansimha/graphql-code-generator#7764] [Q&A]
Good morning everyone :)

I've been working a lot with `graphql-code-generator` lately and it's been a great experience so far, congratulations to the team and all contributors 👏🏻 

I have a small question related to `importAllFragmentsFrom` from the `near-operation-file` preset. In the docs, it says that it can receive a `string` or an `object` but I couldn't find any proper example of the `object` that it should receive. To state my case, I'm working on a `monorepo` and I have a query that imports fragments from different packages and I can tell `importAllFragments: '~@my-app/fragment-one-pkg` and it will work fine, but this query could have fragments from `@my-app/fragment-two-pkg` as it is an architectural decision to split it this way. Is there a way that I can have  `importAllFragmentsFrom` to use multiple packages? :)

Thank you so much in advance :)

**Comment by denjalonso** [comment]
@alexvcasillas did you find any solution?
